cmd/kola/kola.go

```diff
@@ -47,18 +46,19 @@ var (
 )
 
 func init() {
-	cmdRun.Flags().StringVar(&kolaPlatform,
-		"platform", "qemu", "VM platform: qemu or gce")
-	cmdRun.Flags().StringVar(&EtcdRollingVersion,
+	sv := cmdRun.Flags().StringVar
+
+	sv(&EtcdRollingVersion,
```

Comment:
Unless you are going to further reformat this and make all the sv calls a single line I don't think there is much value in using sv instead of the full name.